docs/gallery.rst

```diff
@@ -76,3 +76,5 @@ Gallery
 .. image:: _static/img/viz_thumbnails/horizon.png
    :scale: 25 %
 
+.. image:: _static/img/viz_thumbnails/mapbox.png
```

Comment:
all the other files in that folder have fixed height/width, I forgot the number of pixels but you should just match is already in there or it will look funky in some places (like in the docs gallery where we have less control over html/css)